specification/communication/Communication.Messages/models.tsp

```diff
@@ -0,0 +1,370 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
```

Comment:
Why do we need this import? We should constrain this to the client.tsp file and use augment (e.g. `@@access(CommunicationMessageType, Access.public)`

Reply:
Moved them out of models.tsp, into client.tsp